README.md

```diff
@@ -1,12 +1,13 @@
-# Next Logger [![Circle CI](https://circleci.com/gh/Financial-Times/n-logger.svg?style=svg)](https://circleci.com/gh/Financial-Times/n-logger)
+# Next Logger [![Circle CI](https://circleci.com/gh/Financial-Times/n-logger.svg?style=svg)](https://circleci.com/gh/Financial-Times/n-logger) ![GitHub release](https://img.shields.io/github/release/Financial-Times/n-logger.svg?style=popout)
 
 N-logger is a Winston wrapper which sends logs to Splunk's Http Event Collector (HEC).
 
-## Installation
+N-logger is not only used by Next. Please be mindful of this when making changes, as they may impact other teams, such as Internal Products, differently.
 
-    npm install @financial-times/n-logger
+## Installation
+`npm install @financial-times/n-logger`
```

Comment:
I'd suggest using triple backticks to be consistent with the formatting of the other command / code examples below.